dataland-frontend/src/components/general/AlignedActivitiesDataTable.vue

```diff
@@ -168,13 +170,33 @@ export default defineComponent({
         ...createActivityGroupData<number>(
           col.activityName as string,
           "substantialContributionCriteria",
-          col.substantialContributionCriteria,
-          (value: number) => (value ? `${value}%` : ""),
+          {
+            substantialContributionToClimateChangeMitigation: col.substantialContributionToClimateChangeMitigation,
```

Comment:
Is this construction with col.substantialContributionTo.... necessary? Seems a bit complicated

Reply:
And `substantialContributionToClimateChangeMitigation` should be `substantialContributionToClimateChangeMitigationInPercent` etc.   Right?

Reply:
I think Paul has changed this

Reply:
Not yet:

![image](https://github.com/d-fine/Dataland/assets/99255296/e16c196b-84a3-4ddf-aec1-2bd945d83098)

Reply:
Probably not pushed yet

Reply:
Seems to have been pushed now

Reply:
ok